### ISSUE TITLE
Example Pages Should Be All WCAG 2.0 AA and WAI ARIA conform

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#515]
All example pages should be WCAG 2.0 at least AA and WAI ARIA conform. At the moment they're not and have several accessibility issues. These pages should be built from scratch, so all the Heading and other elements are semantic correctly implemented and accessible. For example, quick accessibility check of "Homepage with global navigation", result:

![image](https://cloud.githubusercontent.com/assets/7510842/22778405/1d77cb48-eeb7-11e6-882a-ab340b9185bb.png)

**Comment by bit-pro-iew-eui** [comment]
Please analyse and estimate @topaxi
